sys-apps/keyutils/files/tmpfiles.d/keyutils.conf

```diff
@@ -0,0 +1,3 @@
+L	/etc/request-key.conf	-	-	-	-	../usr/share/keyutils/request-key.conf
+L	/etc/request-key.d	-	-	-	-	../usr/share/keyutils/request-key.d
```

Comment:
if `/etc/request-key.d` is a directory, i dont see any reason for `/usr/share/keyutils/request-key.d` to be created by the make install. should that be patched out of the makefile?